AT_Training/Exercise_2/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace Exercise_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int L, W, H;
+        Label1: Console.Write("Length:");
```

Comment:
Good job, but this code is pretty hard too read because of many gotos
Should be simplified. We code discuss it (for ex. in voice) how to remove gotos